### PR TITLE
Ts data pipeline

### PR DESCRIPTION
**Pull request by BergsethCognite**
# Description

Please describe the change you have made.

## Checklist

- [ ] Tests added/updated.
- [ ] Run Demo Job Locally.
- [ ] Documentation updated.
- [ ] Changelogs updated in [CHANGELOG.cdf-tk.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.cdf-tk.md).
- [x] Template changelogs updated in [CHANGELOG.templates.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.templates.md).
- [ ] Version bumped.
  [_version.py](https://github.com/cognitedata/cdf-project-templates/blob/main/cognite/cognite_toolkit/_version.py) and
  [pyproject.toml](https://github.com/cognitedata/cdf-project-templates/blob/main/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite_toolkit/cognite_modules/examples/cdf_data_pipeline_timeseries_valhall/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+| timeseries_location_read_group_source_id | Object/ Source ID for security group in IdP. Used to read file data|
+
+## Usage
+
+You should copy and rename an example module into the `custom_modules` directory (remove any `cdf_` prefixes) and make
+your own modifications. You should then update the `selected_modules_and_packages:` section in your `config.[env].yaml`
+file to install the module.
+
+`NOTE: Using Cognite Functions to run workloads will be limited by the underlying resources in the cloud provider functions.
+Hence processing many P&ID documents will not be optimal in a CDF function since it will time out and fail.`
```
[the remark this answers is not in the document]
Comment (BergsethCognite) [reply cognite_toolkit/cognite_modules/examples/cdf_data_pipeline_timeseries_valhall/README.md]: copy paste error - fixed :-)

File: CHANGELOG.templates.md [hunk 2]
```diff
@@ -15,7 +15,14 @@ Changes are grouped as follows:
...
 
-## [0.1.1] - 2024-03-01
+## [0.1.0b9] - 2024-03-01
+
+No changes to templates.
+
+- Added a new module `cognite_modules/example/cdf_data_pipeline_timeseries_valhall` opcua / time series extractor pipeline,
+  CDF function running contextualization of time series to assets.
+
+## [0.1.0b8] - 2024-02-14
```
[the remark this answers is not in the document]
Comment (BergsethCognite) [reply CHANGELOG.templates.md:25]: agree - tried to update

**Review by haakonvt — APPROVED** [review]
LGTM! 🚀

File: CHANGELOG.templates.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (haakonvt) [reply CHANGELOG.templates.md:25]: Afaik, there's two changelog files

[the remark this answers is not in the document]
Comment (BergsethCognite) [reply CHANGELOG.templates.md]: ups, yes updated wrong changelog file
fixed

**Review by ronpal — APPROVED** [review]

**Pull request merged**